config/frontend/config.json

```diff
@@ -16,7 +17,7 @@
    "jsonMetadataEnabled":true,
    "jupyterHubUrl":"",
    "landingPage":"",
-   "lbBaseURL":"",
+   "lbBaseURL":"http://127.0.0.1",
```

Comment:
is it really necessary to add localhost here? for the old back-end `lbBaseURL` used to be empty and thus the configuration could also be used if docker wasn't running on the local machine but remotely. What is the advantage of specifying `http://127.0.0.1`?

Reply:
ok, that's a bit unfortunate since we had more flexibility there before but if there is no other way we will have to live with that.